projects/observability/src/pages/explorer/explorer-service.test.ts

```diff
@@ -0,0 +1,77 @@
+import { fakeAsync } from '@angular/core/testing';
+import { NavigationParamsType } from '@hypertrace/common';
+import { FilterBuilderLookupService, FilterOperator, toUrlFilterOperator } from '@hypertrace/components';
+import { AttributeMetadata, AttributeMetadataType, MetadataService } from '@hypertrace/distributed-tracing';
+import { ExplorerService, ScopeQueryParam } from '@hypertrace/observability';
+import { runFakeRxjs } from '@hypertrace/test-utils';
+import { createServiceFactory, mockProvider } from '@ngneat/spectator/jest';
+import { of } from 'rxjs';
+
+describe('Explorer service', () => {
+  const createService = createServiceFactory({
+    service: ExplorerService
+  });
+
+  const getAttributeMock = jest.fn((scope, attributeKey) =>
+    of(
+      attributeKey === 'duration'
+        ? {
+            name: 'duration',
+            displayName: 'Duration',
+            units: 'ms',
+            type: AttributeMetadataType.Number,
+            scope: scope,
+            requiresAggregation: false,
+            allowedAggregations: [],
+            groupable: false
+          }
+        : {
+            name: 'status',
+            displayName: 'Status',
+            units: '',
+            type: AttributeMetadataType.String,
+            scope: scope,
+            requiresAggregation: false,
+            allowedAggregations: [],
+            groupable: false
+          }
+    )
+  );
+
+  const lookupMock = jest.fn(() => ({
+    buildFilter: (attribute: AttributeMetadata, operator: FilterOperator, value: number) => ({
+      metadata: attribute,
+      field: attribute.name,
+      operator: operator,
+      value: value,
+      userString: '',
+      urlString: `${attribute.name}${toUrlFilterOperator(operator)}${value}`
+    })
+  }));
+
+  test('creates nav params correctly', fakeAsync(() => {
```

Comment:
nit: since we're using the rxjs test scheduler now, it shouldn't need to be fakeAsync any more.

Reply:
@aaron-steinfeld fixed this. reapproval please.